clients/omnitruck/idynamo_services.go

```diff
@@ -0,0 +1,22 @@
+package omnitruck
+
+import (
+	"reflect"
+
+	"github.com/chef/omnitruck-service/models"
+)
+
+type IDynamoServices interface {
+	VersionLatest(params *RequestParams) (ProductVersion, error)
```

Comment:
Can't we just create the interface inside the dynamo_service file as done in the omnitruck.go ?